s4e-backend/src/main/java/pl/cyfronet/s4e/bean/ProductCategory.java

```diff
@@ -24,4 +24,7 @@
 
     @NotEmpty
     private String name;
+
+    @NotEmpty
+    private String iconName;
```

Comment:
Isn't name sufficient in the end?

Reply:
No, because of the repetition of used icons